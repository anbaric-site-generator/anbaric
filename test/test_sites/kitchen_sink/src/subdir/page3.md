---
template: null
---
# This is subdir/page3
