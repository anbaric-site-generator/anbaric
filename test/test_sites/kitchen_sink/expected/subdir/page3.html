<html>
  <head></head>
  <body>
    <h1>This is subdir/page3</h1>
  </body>
</html>